modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -229,6 +229,17 @@
 	}
 }
 
+// TODO: Remove once PHP global color support is fixed.
+.elementor .elementor-element.elementor-widget-n-tabs > .elementor-widget-container > .e-n-tabs > .e-n-tabs-heading .e-n-tab-title {
+	&.e-active {
+		background: var( --n-tabs-title-background-color-active );
```

Comment:
Could it work if we use `background-color` here and then add `background-image: initial;`?

Reply:
Changed to your suggestion, 
Looks like it works OK.